thejoker/sampler/mcmc.py

```diff
@@ -56,7 +56,7 @@ def to_mcmc_params(cls, p):
         .. math::
 
             \ln P \\
-            \sqrt{K}\,\cos\phi_0, \sqrt{K}\,\sin\phi_0 \\
+            \sqrt{K}\,\cos(M_0-\omega), \sqrt{K}\,\sin(M_0-\omega) \\
```

Comment:
We are `M_0 - \omega`, but Murray & Correira imply that it should be `M_0 + \omega`: is this a manifestation of a sign issue I realized long ago and forgot about? Probably.

Reply:
I think we should track down that sign issue so we know what it is. I feel like it might have to do with inclination choices (are we looking at the top-side or bottom-side of the orbit?).